test/auth.test.ts

```diff
@@ -0,0 +1,78 @@
+import request from 'supertest';
+import { server } from '../src/index';
+import { app } from '../src/app';
+import prisma from '../src/utils/client';
+
+const req = request(app);
+interface IUserInfo {
```

Comment:
You shouldn't create the type from scratch. Instead you should use the type inferred by your validation schema.